pom.xml

```diff
@@ -220,12 +240,12 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>3.0.0-M3</version>
+                <version>3.0.0</version>
```

Comment:
Did you test this upgrade? When I tried this, I get a parse configuration error during build/test.

Reply:
1.4+ resolved the issues.